app/timetables/models.py

```diff
@@ -39,3 +39,26 @@ def clean(self):
 
     def __str__(self):
         return self.name
+
+
+class Timetable(ForceCapitalizeMixin, TimestampMixin):
+    """
+    A timetable is the central entity or model of the platform. 
+    
+    It represents/encapsulates the entire structure and 
+    scheduling of meals, menu-items, dishes, courses, options etc, 
+    served at a location, to a team or the entire organisation.
+
+    Timetable model has a many-to-many relationship with the Weekday model.
+    """
```

Comment:
@andela-joyebanji I do not think you need to specify the relationship in the docstring.